BrainPortal/app/views/userfiles/show.html.erb

```diff
@@ -270,7 +270,11 @@
           <div class="display_userfile">
             <% begin %>
               <% if @viewer %>
-                <%= render :file  => @viewer.partial_path %>
+                <% if @viewer.errors.empty? %>
+                  <%= render :file  => @viewer.partial_path %>
+                <% else %>
+                  <%= render :partial  => "viewer_errors" %>
```

Comment:
I don't understand how it's possible that we have this partial rendered both here and in the controller?